src/components/GetDataDialog.tsx

```diff
@@ -19,45 +26,103 @@ function GetDataDialog({
   disableDownloadResultsButton: boolean;
   handleDownloadResultButtonClick: (identifier: string) => void;
 }) {
+  const DOCKER_RUN_COMMAND =
+    'docker run -t -v $(pwd):/data neurobagel/dataget:latest /data/cohort-participant-machine-results.tsv /data/output';
   const theme = useTheme();
   const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
 
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
+  const [showPopover, setShowPopover] = useState(false);
+
+  const handleCopyClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    navigator.clipboard.writeText(DOCKER_RUN_COMMAND);
+    setAnchorEl(event.currentTarget);
+    setShowPopover(true);
+
+    setTimeout(() => {
+      setShowPopover(false);
+    }, 2000);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+    setShowPopover(false);
+  };
+
   return (
     <Dialog fullScreen={fullScreen} open={open} onClose={onClose} data-cy="get-data-dialog">
-      <DialogTitle>Example usage</DialogTitle>
       <DialogContent>
         <DialogContentText>
-          Please follow the steps below
+          <Typography variant="h6" className="font-bold">
+            Query results for programmatic use/download
```

Comment:
🍒 looking at this again I think there might be a more intuitive phrasing still. How about:
```suggestion
            Results file for data access or programmatic use
```